frontend/src/actions/ViewActions.js

```diff
@@ -222,6 +223,28 @@ export function addViewLocationData(id, locationData, isModal) {
   };
 }
 
+/**
+ * @method toggleIncludedView
+ * @summary success when updating view's properties
+ */
+export function updateViewSuccess(id, data, isModal) {
```

Comment:
Can you please put params in an object updateViewSuccess({ id, data, isModal }) {